main/src/ca/uwaterloo/flix/tools/pkg/FlixPackageManager.scala

```diff
@@ -90,6 +105,90 @@ object FlixPackageManager {
     }
   }
 
+  //TODO: test!
```

Comment:
@magnus-madsen could you add a Maven dependency to helloworld so I can see if it is found with hellouniverse? I have used this one before for testing: "org.junit.jupiter:junit-jupiter-api" = "5.9.2"

Reply:
Done, but I added it to hellouniverse instead.

Reply:
Perhaps, in a short while, we can draw a little dependency tree on paper, and then re-create that in GitHub.